src/psyclone/tests/test_files/gocean1p0/test27_loop_swap.f90

```diff
@@ -0,0 +1,58 @@
+!-------------------------------------------------------------------------------
```

Comment:
I suspect that you can cut this example down a lot - think of the bare minimum a compiler would accept (even if the compiled code would not execute). We support compiling code as part of the test suite for LFRic and ideally we'll extend this to the GOcean API. Therefore, any test code should be compilable (but not necessarily linkable).